bundle/edu.gemini.phase2.skeleton.servlet/src/main/resources/edu/gemini/phase2/template/factory/xml/GNIRS_BP.xml

```diff
@@ -54,8 +55,15 @@ The semicircular portion of the field is below the rectangle. The template scien
           <param name="priority" value="MEDIUM"/>
           <param name="tooOverrideRapid" value="false"/>
           <param name="phase2Status" value="INACTIVE"/>
+          <paramset name="schedulingBlock">
+            <param name="start" value="1588354200000"/>
+            <paramset name="duration">
+              <param name="tag" value="unstated"/>
+            </paramset>
+          </paramset>
```

Comment:
I'm concerned about these scheduling blocks being inserted into library observations.  These are supposed to work for any semester.  This may have been an oversight resulting from opening up the library program in the OT and saving it?

Reply:
I brought up your comment with @andrewwstephens and he replied:

"I suggest we just try it and see if the instantiated observations look okay... I see the time is 2020-05-01 17:30:00 so we should notice a problem right away.  We will test it with 20B programs which should get default scheduling dates of ~October"

He suggests merging and seeing if an error crops up in the test release.

Reply:
I think it would be a good idea to test it out with a proposal yourself anyway before merging if you haven’t yet.